catalogue/aarch64-ifetch/shelf.py

```diff
@@ -10,11 +10,12 @@
 
 illustrative_tests = [
     "tests/DIC0-IDC0/MP-inst+dc.cvau-dsb.ish-ic.vau-dsb.ish.litmus",
-    "tests/DIC0-IDC0/MP.RF+cachesync+ctrlisb",
+    "tests/DIC0-IDC0/MP.RF+cachesync+ctrlisb.litmus",
     "tests/DIC0-IDC0/SM.B+cachesync-isb.litmus",
     "tests/DIC0-IDC0/coFF+cachesync.litmus",
     "tests/DIC0-IDC0/MP.RF+dc-dsb+ctrlisb-ic-dsb-isb.litmus",
-    "tests/DIC0-IDC1/IDC1.MP.RF+dmb.st+ctrlisb-ic-dsb-isb.litmus ",
-    "tests/DIC1-IDC1/IDC1.MP.RF+dmb.st+ctrlisb-ic-dsb-isb.litmus ",
+    "tests/DIC0-IDC1/IDC1.MP.RF+dmb.st+ctrlisb-ic-dsb-isb.litmus",
+# This test does not exist.
+#    "tests/DIC1-IDC1/IDC1.MP.RF+dmb.st+ctrlisb-ic-dsb-isb.litmus",
```

Comment:
Hi @artkhyzha, I neutralised this test as there is no test with that name.

Reply:
```suggestion
#    "tests/DIC1-IDC1/DIC1.MP.RF+dmb.st+ctrlisb-ic-dsb-isb.litmus",
```

Ah, a copy-and-paste error on my part; this suggestion could fix the typo.